src/data/biomes/chargestone-cave.ts

```diff
@@ -0,0 +1,146 @@
+import { Biome } from "#app/data/biome";
+import { BiomeId } from "#enums/biome-id";
+import { BiomePoolTier } from "#enums/biome-pool-tier";
+import { SpeciesId } from "#enums/species-id";
+import { TerrainType } from "#enums/terrain-type";
+import { TimeOfDay } from "#enums/time-of-day";
+import { TrainerType } from "#enums/trainer-type";
+import { WeatherType } from "#enums/weather-type";
+
+/**
+ * TODO: Add original art and audio assets
+ */
+
+const pokemonPool = {
```

Comment:
Don't we have interfaces at the ready for those? So the compiler would catch it if something was missing?